torch_geometric/distributed/local_feature_store.py

```diff
@@ -11,6 +11,21 @@
 from torch_geometric.data.feature_store import _FieldStatus
 from torch_geometric.typing import EdgeType, NodeType
 
+from .rpc import RpcCallBase, RpcRouter, rpc_register, rpc_request_async
+
+
+class RpcCallFeatureLookup(RpcCallBase):
```

Comment:
Can we rename to `RPCCallFeatureLookup` and `RPCCallBase`?

Reply:
done